src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const changeProperties = {};
+
+  for (const key in robot) {
+    if (changeProperties.hasOwnProperty(key)
+    || changeProperties.hasOwnProperty(robot[key])) {
+      return null;
+    }
+
+    if (!changeProperties.hasOwnProperty(key)
+    || changeProperties.hasOwnProperty(robot[key])) {
+      changeProperties[robot[key]] = key;
+    }
```

Comment:
So, you can delete these checks, bc you do it before
```suggestion
      changeProperties[robot[key]] = key;
```